pkg/cloud/cloud.go

```diff
@@ -79,6 +89,7 @@ type Cloud interface {
 	DeleteAccessPoint(ctx context.Context, accessPointId string) (err error)
 	DescribeAccessPoint(ctx context.Context, accessPointId string) (accessPoint *AccessPoint, err error)
 	DescribeFileSystem(ctx context.Context, fileSystemId string) (fs *FileSystem, err error)
+	DescribeMountTargets(ctx context.Context, fileSystemId, az string) (fs *MountTarget, err error)
```

Comment:
What's the difference between this DescribeMountTargets and the DescribeMountTargets in efs-utils itself? Is this one used to retrieve mounttargetip address and use the mounttargetip option to mount?

Reply:
There is no difference, they are the same APIs.